hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy

```diff
@@ -151,6 +157,68 @@ class KafkaConsumerGroupManagerSpec extends Specification {
         kafkaContainer.dockerClient.unpauseContainerCmd(containerId).exec()
     }
 
+    def "Should delete specified consumer group and retain others"() {
```

Comment:
Nice. Can we also add some integration test for that mechanism? I mean the one which is creating topic and subscription (with validation of creation of consumer-group), deleting it and then checking whether consumer group was deleted too. What do You think about it?

Reply:
sure, good idea